### DISCUSSION TITLE
https://iptv-org.github.io/iptv/index.m3u keeps crashing

### DISCUSSION THREAD

**Discussion by samg80** [iptv-org/community#262] [Show and tell]
Greetings,
First of all, I wanted to thank you all for this great link and contribution to free tv.
I have been using this M3u list -https://iptv-org.github.io/iptv/index.m3u- for awhile and I really like it, however, recently, I have been having an issue when trying to play a channel. 
I usually go to the channel list and then add the ones that I'd like to watch to my Favorite list. Sometimes, when I go to my Favorites, and even the Channel list and I try to play the channel - for example, I like to watch DW- when I click, it crashes my Kodi ( I am on Kodi 19.1 Matrix)
I restart my Kodi and let the channels sync w/the backend, however, it still crashes again and does not play.
Please advise how to troubleshoot this issues.

Thanks

**Comment by keystroke3** [comment]
Please post the kodi logs in a github gist and link it here

**Reply by samg80** [reply]
@keystroke3 Since I am running Kodi on my FireTV, it is hard to download the logs; this is the only log that I was able to collect. Hope it helps
https://www.dropbox.com/s/igtganzug23zktx/2021-07-17%2021.24.42.mp4?dl=0
Note: I cleared the logs and tested to see which logs were for that specific crash, I saw it was not generating a log when crashing - maybe I am missing something.

**Reply by keystroke3** [reply]
There doesn't seem to be anything useful that points to why the kodi is crashing.If you can find the log file, delete it first, then completely close kodi. Open it again and then reproduce the steps to get the error. You can the post the new log file you get. If you have flash drive, you can copy the log file to it and place it  on a github gist. If you don't, you can connect your phone to the TV and copy the log file to the phone and post it. On android, the various paths to the log file are:
``` 
Windows: C:\Users\{your Windows username}\AppData\Roaming\Kodi\kodi.log
Mac: /Users/{your username}/Library/Application Support/Kodi/kodi.log
Linux: ~/.kodi/kodi.log
iOS: /private/var/mobile/Library/Preferences/Kodi/kodi.log
Android: {storage}/Android/data/org.xbmc.kodi/files/.kodi/temp/kodi.log
OSMC: /home/osmc/.kodi/temp/kodi.log
```

**Reply by samg80** [reply]
I did actually check the logs in this directory also, did not see any -Android: {storage}/Android/data/org.xbmc.kodi/files/.kodi/temp/kodi.log
I think you can close this case for now, I am going to remove the m3u and add another one, see if I will have the same issue. If so, I will reopen the case.
Thanks for your help.
